### PR TITLE
Tidy up installation, add Vagrantfile

### PR DESCRIPTION
**Pull request by devcurmudgeon**
First pass at install_dependencies.sh, and now we ship plantuml.jar

### CODE REVIEW

**Comment by devcurmudgeon** [comment]
@jmacarthur are you ok to merge this?

**Comment by jmacarthur** [comment]
I left an issue on the previous version which, and the new version also has a git conflict marker added to the README.md

**Comment by devcurmudgeon** [comment]
oops, sorry. the conflict should be fixed now

**Comment by jmacarthur** [comment]
Thanks, any comments about the shared folder issue? Vagrant won't start following the instructions you have at the moment.

**Comment by devcurmudgeon** [comment]
eek. I can't see the shared folder issue. i guess my force-pushes to this branch may have lost it :/

**Comment by jmacarthur** [comment]
My mistake, I think - adding a comment to a file starts a review, but that review is only visible to me until I submit it, apparently.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Vagrant.configure(2) do |config|
+  config.vm.box = "ubuntu/zesty64"
+  config.vm.network "private_network", type: "dhcp"
+  config.vm.synced_folder ".", "/src/mustard"
+  config.vm.synced_folder "../content", "/src/content"
```
Comment (devcurmudgeon) [inline Vagrantfile]: Thanks for the review.

I think it needs to be both, but your comment made me realise that the directory needs to be created, not just shared and cloned. I've pushed another version

**Comment by jmacarthur** [comment]
OK, that looks better - I haven't been able to fully test the vagrant file, but I think that's to do with my setup, rather than this patch, so I'm happy to merge it. Thanks!

**Pull request merged**